WebApp/autoreduce_webapp/static/javascript/tours/instrument_summary_tour.js

```diff
@@ -0,0 +1,64 @@
+steps = [
+    {
+        element: "#not-active",
```

Comment:
This works only when the instrument is not active. We should change this step to say instrument is not active - I am actually going to remove this from the db soon as we don't use it so no need to elaborate.